Automattic/jetpack
# Jetpack API package

Moves shared internal API functions to the `@automattic/jetpack-api` package.

This is part of work on Jetpack RNA p2-p9dueE-2sp.

#### Changes proposed in this Pull Request:

* Move rest-api class into a shared js-package

#### Jetpack product discussion
p2-p9dueE-2sp

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Build, connect Jetpack and confirm that everything in the dashboard loads correctly.
* Follow instructions below to run tests for the new package

```sh
cd projects/js-packages/api
pnpm run test
```

Right now there are minimal tests, just there to act as a guide as we add and modify API methods.
